router/pump.go

```diff
@@ -73,6 +73,16 @@ func ignoreContainer(container *docker.Container) bool {
 	if value, ok := container.Config.Labels[excludeLabel]; ok {
 		return len(excludeLabel) > 0 && strings.ToLower(value) == "true"
 	}
+	includeLabel := getopt("INCLUDE_LABEL", "")
+	if includeLabel != "" {
+		if _, ok := container.Config.Labels[includeLabel]; ok {
```

Comment:
Is there a reason this check isn't similar to the exclude check where we check the actual value?